railway-deploy.sh

```
@@ -18,17 +18,54 @@ railway link
+    export $(grep -v '^#' .env | xargs)
+fi
+
+# Function to prompt for a variable if it is not set
+prompt_var() {
+    local var_name=$1
+    local is_secret=$2
+    local current_val="${!var_name}"
+
+    if [ -z "$current_val" ]; then
+        if [ "$is_secret" = "true" ]; then
+            read -s -p "Enter $var_name: " user_input
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)